packages/myst-cli/src/store/reducers.ts

```diff
@@ -39,18 +39,13 @@ export const config = createSlice({
   } as {
     currentProjectPath: string | undefined;
     currentSitePath: string | undefined;
-    rawConfigs: Record<string, Record<string, any>>;
+    rawConfigs: Record<string, { raw: Record<string, any>; validated: Record<string, any> }>;
     projects: Record<string, ProjectConfig>;
     sites: Record<string, SiteConfig>;
     siteTemplateOptions: Record<string, SiteTemplateOptions>;
     filenames: Record<string, string>;
   },
   reducers: {
-    reload(state) {
```

Comment:
Every time we `loadConfigs`, we compare the raw config to the cached raw config and, if it has changed, reload all the site/project configs.

This means we do not need to clear out the state like this; reloading will replace outdated configs.